stestr/config_file.py

```diff
@@ -62,8 +62,10 @@ def get_run_command(self, options, test_ids=None, regexes=None):
             top_dir = options.top_dir
         elif self.parser.has_option('DEFAULT', 'top_dir'):
             top_dir = self.parser.get('DEFAULT', 'top_dir')
-        command = "${PYTHON:-python} -m subunit.run discover -t" \
-                  " %s %s $LISTOPT $IDOPTION" % (top_dir, test_path)
+
+        python = 'python' if sys.platform == 'win32' else '${PYTHON:-python}'
```

Comment:
So, this still works, assuming that you have python's dir in the PATH env variable (which is pretty much always).

How you access an environment variable on Windows depends on what you're using: cmd, or powershell: http://pastebin.com/gSh94kw1
The env variable for python python is PYTHONPATH. We could probably have something like this:

    if sys.platform != 'win32':
        python = '${PYTHON:-python}'
    else:
        python = os.getenv('PYTHONPATH')
    ...